.github/workflows/ci.yml

```diff
@@ -30,20 +30,33 @@ jobs:
         run: cargo clippy --all-targets --locked
 
       - name: Build
-        run: deno task node
+        run: deno task build && deno task node
 
       - name: Test
         run: |
           cargo test --all-targets --locked
           deno task test
 
-      - name: Check Wasm up to date
-        run: deno task build --check
+      - name: Get tag version
+        if: startsWith(github.ref, 'refs/tags/')
+        id: get_tag_version
+        run: echo ::set-output name=TAG_VERSION::${GITHUB_REF/refs\/tags\//}
+
+      - name: Publish deno.land/x
+        uses: denoland/publish-folder@82ce065074e7174baf444332c4b7c40869a4909a
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          folder: js
+          branch: deno_registry
+          tag: deno/${{ steps.get_tag_version.outputs.TAG_VERSION }}
```

Comment:
Todo: update the version prefix webhook in this repo.

Reply:
Is this supposed to be addressed after the merge?

Reply:
Yes.